src/ping/handler.js

```diff
@@ -1,48 +1,46 @@
 'use strict'
 
-const pull = require('pull-stream/pull')
-const handshake = require('pull-handshake')
-const constants = require('./constants')
-const PROTOCOL = constants.PROTOCOL
-const PING_LENGTH = constants.PING_LENGTH
+const pipe = require('it-pipe')
+const handshake = require('it-handshake')
+const { PROTOCOL } = require('./constants')
 
 const debug = require('debug')
 const log = debug('libp2p-ping')
 log.error = debug('libp2p-ping:error')
 
-function mount (swarm) {
-  swarm.handle(PROTOCOL, (protocol, conn) => {
-    const stream = handshake({ timeout: 0 })
-    const shake = stream.handshake
+/**
+ * Subscribe ping protocol handler.
+ * @param {Libp2p} node
+ */
+function mount (node) {
```

Comment:
@jacobheun what do you think about these namings?

I thought that we could make this more consistent with the transports and have a `createListener` function or similar. But, I decided to gather your feedback first.

Reply:
I think mount/unmount is okay. We're not so much creating listeners as we are attaching an application protocol to libp2p. Different services/protocols may have different needs so keeping it generic might be good. If we ever create an interface for this type of service we could reevaluate this then.

Reply:
agreed